swarm/src/behaviour.rs

```diff
@@ -320,6 +320,10 @@ pub enum ToSwarm<TOutEvent, TInEvent> {
         /// Whether to close a specific or all connections to the given peer.
         connection: CloseConnection,
     },
+
+    /// TODO: more documentation needed
+    /// Reports external address of a remote peer to other `NetworkBehaviours`
+    NewExternalAddrOfPeer { addr: Multiaddr, peer_id: PeerId },
```

Comment:
Intuitively, I'd have expected these to be flipped, i.e. for the `PeerId` to be at the front.

Reply:
:heavy_check_mark: